tools/blocksconvert/builder/tsdb.go

```diff
@@ -297,20 +301,61 @@ func addSeriesToIndex(indexWriter *index.Writer, sl *seriesList, unsortedChunksR
 		return stats, errors.Wrap(err, "reading series")
 	}
 
-	ix := 0
+	type chunkToRead struct {
+		ref   uint64
+		chunk *chunkenc.Chunk
+		err   *error
+	}
+
+	ch := make(chan chunkToRead)
```

Comment:
Would help performances if we add a small buffer to the channel?

Reply:
I don't see how it would help the performance. Can you elaborate?

Reply:
No worries, it's OK as is.